src/voluntarily.yaml

```diff
@@ -0,0 +1,49 @@
+---
+repoUrl: 'https://github.com/voluntarily/vly2'
+name: Voluntarily
+description: >-
+  Platform for volunteers and projects to find each other. Volunteers can
+  include time, advice or physical things.
+websiteUrl: 'http://voluntarily.nz'
```

Comment:
I'd avoid linking to http websites. There's a redirection to `https://live.voluntarily.nz/`, so how about replacing with that link instead?